bin/GameIndex.yaml

```diff
@@ -2681,6 +2675,8 @@ SCKA-20011:
   region: "NTSC-K"
   gameFixes:
     - VU0KickstartHack # Fixes Character SPS.
+    - EETimingHack # Fixes DMA errors 
```

Comment:
This timing hack needs to be removed.

Reply:
Have you played through the Korean version to make sure it isn't required?

Reply:
As long as the game is NTSC it will not have the problem. PAL have some very specific code that introduce a DMAC glitch.